src/components/common/wrappers/footer-wrapper.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from "react";
+import { Outlet, matchPath, useLocation } from "react-router-dom";
+
+import { Footer } from "~/components/Footer/Footer.tsx";
+import { AppRoute } from "~/libs/enum/index.ts";
+
+const NoFootersRoutes = [AppRoute.ROOT, AppRoute.SIGN_UP, AppRoute.SIGN_IN];
+const GetStartedFooterRoutes = [
+	AppRoute.TERMS_OF_USE,
+	AppRoute.HOME,
+	AppRoute.PRIVACY_POLICY,
+];
```

Comment:
Here the code is duplicated in two components. In my opinion, we can use them like a joint const from one file.

Reply:
agree